Makefile

```diff
@@ -1,4 +1,4 @@
-DEV_ROCKS = busted luacheck lua-llthreads2
+DEV_ROCKS = "busted 2.0.rc12" "luacheck 0.20.0" "lua-llthreads2 0.1.4"
```

Comment:
This won't work as is — I mean, it will install all rocks on first execution, but the whole logic in the `dev` rule to avoid re-installations on subsequent executions will not.

The `grep` arguments are not quoted, which makes `grep` fail every time, and the output of `luarocks list` does not print "<package> <version>" like that anyway.

To use the output of `luarocks` programmatically in scripts, use the `--porcelain` option, which produces a stable script-friendly output (output then becomes "<program>\t<version>").

I'll push a fix to this PR.

Reply:
thx. good catch!